clickhouse/client.cpp

```diff
@@ -161,6 +161,12 @@ class Client::Impl {
 
     void Insert(const std::string& table_name, const std::string& query_id, const Block& block);
 
+    PreparedInsert * PrepareInsert(Query query);
+
+    void FinishInsert();
+
+    void SendData(const Block& block);
```

Comment:
I had to move this to public so that PreparedInsert can call it. Not in the header file, though, so shouldn't matter.